database/db_build.sql

```diff
@@ -0,0 +1,35 @@
+BEGIN ;
+
+DROP TABLE IF EXISTS users CASCADE;
+DROP TABLE IF EXISTS posts CASCADE;
+
+CREATE TABLE users (
+user_id SERIAL PRIMARY KEY ,
+user_name VARCHAR(100) NOT NULL UNIQUE ,
+password VARCHAR(100) NOT NULL ,
+email TEXT NOT NULL UNIQUE,
+country TEXT NOT NULL,
+city TEXT NOT NULL,
+added_date DATE DEFAULT now()
+);
+
+CREATE TABLE posts(
+  post_id SERIAL PRIMARY KEY,
+  user_id INTEGER NOT NULL ,
+  prefferedPayment VARCHAR(100) NOT NULL,
+  country TEXT NOT NULL,
+  city TEXT NOT NULL,
+  coin TEXT NOT NULL,
+  qty INTEGER NOT NULL,
```

Comment:
im guessing this is quantity?? why not have `quantity` just written there

Reply:
we like qty more because we render it.